README.adoc

```diff
@@ -46,6 +46,37 @@ gem install rouge
 ----
 ====
 
+== A Living Document
+
+Per the comment above, this guide is a work in progress - some rules are simply lacking thorough examples, but some things in the Minitest world change week by week or month by month.
+With that said, as the standard changes this guide is meant to be able to change with it.
+
+== Layout
+
+=== Empty Line after test case[[empty-line-after-test-case]]
+
+Add empty lines after each `test`. It make the code more readable.
```

Comment:
I think that "`test_`" or "test methods" is closer to the official term than `test`.
https://github.com/seattlerb/minitest#unit-tests

Reply:
Ok. I'll update it to test methods. 😊

Reply:
@koic Done.